examples/palma.ini

```diff
@@ -26,7 +26,7 @@ id = ":1"
 ;~ ssh = "ssh palma@localhost";
 
 [general]
-; Enable or disable password authorization (default: true).
+; Enable or disable password authorization (default: false).
```

Comment:
If nothing is set here, the code in `DBConnector.class.php` will require password authorization, so this change does not look correct.